ChattoAdditions/Source/Input/ChatInputBar.swift

```diff
@@ -170,6 +173,10 @@ open class ChatInputBar: ReusableXibView {
     public func setTextViewPlaceholderAccessibilityIdentifer(_ accessibilityIdentifer: String) {
         self.textView.setTextPlaceholderAccessibilityIdentifier(accessibilityIdentifer)
     }
+
+    public func setTextViewPlaceholderText(_ text: String) {
```

Comment:
Can you make it computed property, as `inputText`?

Reply:
amending this PR as it will be for swift-3 branch. Fixed in there: https://github.com/badoo/Chatto/pull/396